src/main/java/com/redhat/cloud/policies/app/rest/PolicyCrudService.java

```diff
@@ -885,12 +911,34 @@ public Response getTriggerHistoryForPolicy(@PathParam("id") UUID policyId) {
 
          int limit = pager.getLimit();
          // We dont allow unlimited or values > 200
-         limit = limit == Pager.NO_LIMIT ? 50 : min(limit, 200);
+         limit = limit == Pager.NO_LIMIT ? 50 :  min(limit,200);
          int pageNum = pager.getOffset() / limit;
 
+         // Once Quarkus supports MP rest client 2.0 we can use
+         // String[] here and use some special query-param style to
+         // support this
+         String sort1=null;
+         String sort2 = null;
+         String order1 = null;
+         String order2= null;
+
+         if (pager.getSort()!= null) {
+           List<Sort.Column> columns = pager.getSort().getColumns();
+           if (columns.size() > 0) {
+             sort1 = getSortFromPageColumn(columns.get(0));
+             order1 = getOrderFromPageColumn(columns.get(0));
+           }
+           if (columns.size() > 1) {
+             sort2 = getSortFromPageColumn(columns.get(0));
+             order2 = getOrderFromPageColumn(columns.get(0));
```

Comment:
I suppose you mean the second element
```suggestion
             sort2 = getSortFromPageColumn(columns.get(1));
             order2 = getOrderFromPageColumn(columns.get(1));
```

Reply:
Yeah